model/build.gradle.kts

```diff
@@ -26,7 +26,8 @@ java {
 }
 
 group = "fr.insee.ddi"
-version = "0.1.0-SNAPSHOT"
+version = "1.0.0"
+val nameForArtifactAndJar by extra("ddi-lifecycle")
```

Comment:
I don't know kotlin/gradle enough: I'm curious about this "by extra"

Reply:
I picked it from this example : https://docs.gradle.org/current/userguide/writing_build_scripts.html#sec:extra_properties